docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst

```diff
@@ -349,68 +349,98 @@ State and Transition Data Tables
      - TBL cancer sequelae prevalence from GBD 2019
      - Not forecasted
    * - MST
-     - 2.06 years (95% CI: 0.42 - 3.83); normal distrbution of uncertainty at draw level
+     - Uniform distribution between 2.06 and 5.38 years
      - Mean sojourn time; duration of time between onset of the CT screen-detectable preclinical phase to the clinical phase
      - See below for instructions on how to sample and research background. NOTE: may update this value
-   * - AST
-     - :math:`1/(ACMR - csmr_\text{c426} + \frac{csmr_\text{c426}}{prevalence_\text{C, general population}})`
-     - Average survival time; mean duration of time between detection and death
-     - ACMR: all cause-mortality rate for demographic group from GBD
    * - ODF
-     - 0.35 (0.2, 0.5); normal distribution of uncertainty at the draw level
+     - Triangle distribution of uncertainty with min=4, point=10, max=23. np.random.triangular(4, 10, 23)
```

Comment:
I'm a little confused by the change from values on the order of 0.35 to numbers on the order of 10.